### PR TITLE
[feature] Support OpenBSD

### PR DESCRIPTION
**Pull request by trombik**
this is a ID-less version of #2.

### CODE REVIEW

**Comment by trombik** [comment]
diff: https://github.com/reallyenglish/ansible-role-argus-radium/compare/support_openbsd...support_openbsd_without_ugids

File: README.md [hunk 1]
```diff
@@ -1,24 +1,181 @@
...
 
-| variable | description | default |
+| Variable | Description | Default |
 |----------|-------------|---------|
+| `argus_radium_user` | user of `radium` | `{{ __argus_radium_user }}` |
+| `argus_radium_user_home` | home of the `argus_radium_user` | `{{ __argus_radium_user_home }}` |
+| `argus_radium_user_shell` | shell of `argus_radium_user` | `{{ __argus_radium_user_shell }}` |
+| `argus_radium_user_comment` | GECOS of `argus_radium_user` | `argus radium daemon` |
+| `argus_radium_group` | group of `radium` | `{{ __argus_radium_group }}` |
+| `argus_radium_group_gid` | GID of `argus_radium_group` | `{{ __argus_radium_group_gid | default() }}` |
```
Comment (mitsururike) [inline README.md]: argus_radium_group_gid is not used in the role.

File: vars/FreeBSD.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+__argus_radium_user: radium
+__argus_radium_user_home: /var/log/argus
+__argus_radium_user_shell: /usr/sbin/nologin
+__argus_radium_group: argus
 __argus_raduim_package: net-mgmt/argus3-clients
 __argus_radium_conf_dir: /usr/local/etc
+__argus_radium_conf_file: "{{ __argus_radium_conf_dir }}/radium.conf"
 __argus_radium_flags: "-f {{ __argus_radium_conf_file }}"
```
Comment (mitsururike) [inline vars/FreeBSD.yml]: s/__argus_radium_conf_file/argus_radium_conf_file/
or users must modify argus_radium_flags when argus_radium_conf_file is set.

File: defaults/main.yml [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
-argus_radium_log_dir: "{{ argus_log_dir }}"
+argus_radium_log_dir: /var/log/argus
+argus_radium_log_dir_owner: "{{ argus_radium_user }}"
+argus_radium_log_dir_group: "{{ argus_radium_group }}"
+argus_radium_log_dir_mode: "0755"
 argus_radium_service: radium
 argus_radium_conf_dir: "{{ __argus_radium_conf_dir }}"
-argus_radium_conf_file: "{{ argus_radium_conf_dir }}/radium.conf"
-argus_radium_flags: "-f {{ argus_radium_conf_file }}"
+argus_radium_conf_file: "{{ __argus_radium_conf_dir }}/radium.conf"
```
Comment (mitsururike) [inline defaults/main.yml]: s/__argus_radium_conf_dir/argus_radium_conf_dir/
or users must modify argus_radium_conf_file when argus_radium_conf_dir is set.

File: vars/FreeBSD.yml (hunk 2 above)
Comment (mitsururike) [reply vars/FreeBSD.yml]: vars/RedHat.yml has the same problem.

**Review by mitsururike — APPROVED** [review]

**Pull request merged**